src/course-search/styles/_FacetList.scss

```diff
@@ -40,4 +40,35 @@
       }
     }
   }
+
+  .typeahead {
+    .dropdown-menu {
+      .facet-list & {
+        padding: 50px 0 0;
+        max-height: none;
+        overflow: visible;
+        width: 420px;
+      }
+
+      .typeahead-dropdown-menu-scrollable-items {
+        position: relative;
+        max-height: 200px;
+        overflow: auto;
+
+        .no-options-found {
+          display: block;
+        }
+      }
+
+      .typeahead-dropdown-input {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 98%;
+        background: white;
+        padding: 5px;
+        margin: 4px;
```

Comment:
Typically, we'd should be using the spacing tokens provided by Bootstrap and the Paragon theme for padding/margin styles, e.g., "m-2" and "p-3".

Is there an explicit need to use hardcoded spacing values here?

Reply:
bad habits :( 
Let me look into this.